packages/material-ui/src/Slide/Slide.js

```diff
@@ -35,23 +37,37 @@ function getTranslateValue(direction, node) {
   }
 
   if (direction === 'left') {
+    if (targetNode) {
+      return `translateX(${targetNodeRect.right}px) translateX(${offsetX - rect.left}px)`;
+    }
+
     return `translateX(${containerWindow.innerWidth}px) translateX(${offsetX - rect.left}px)`;
```

Comment:
Why not?

```suggestion
    return `translateX(${containerWindow.innerWidth + offsetX - rect.left}px)`;
```

x the other?

It seems to be from a legacy constraint #8486, we couldn't mix units: `translateY(10vh + 2px)` is KO.